### ISSUE TITLE
404 error when requesting base url after route:trans:cache command

### ISSUE THREAD

**Issue by rene0399** [mcamara/laravel-localization#653]
Hi 
When you request the base url without the locale, it returns a 404 page if you run the route:trans:cache command
steps to recreate:

Fresh install of Laravel 6
install package and run the vendor:publish command
follow the steps to use route:trans:cache command

add the following to the web.php routes:

```
Route::group(
    [
    'prefix' => LaravelLocalization::setLocale(),
    'middleware' => [ 'localeSessionRedirect', 'localizationRedirect', 'localeViewPath' ]
    ], function() {
    /** ADD ALL LOCALIZED ROUTES INSIDE THIS GROUP **/
    Route::get('/','HomeController@index');

});
```

remove unserializable content from api.php routes.

HomeController@index :
```
public function index()
    {
        return app()->getLocale();
    }
```

Run in console:
```
php artisan route:trans:cache
```

**Comment by rene0399** [comment]
Related to #650 
But is not fixed by #651

**Comment by mcamara** [comment]
Same as #650 , fixed in the new version
